### DISCUSSION TITLE
Having ability to create prompts with variables

### DISCUSSION THREAD

**Discussion by tmaier** [AllYourBot/hostedgpt#346] [General]
Similar to this: https://x.com/alexalbert__/status/1788961823284506777?s=46

**Comment by keithschacht** [comment]
Yea, one weak thing about ChatGPT is the suggested questions when you start a new conversation. No one ever uses them; they’re worthless.

I recently played with Magic School. It’s a ChatGPT for teachers. They redesigned this start screen to be filled with a prompt library of really useful tasks. As you look at all the options you think “oh that’s cool I never thought about using it for that.” And then when you click them it doesn’t simply fill in the text input with some predefined text, instead it takes you to a screen asking you to fill in the input variables for predefined prompts. It’s very much like what you linked to, just more consumer friendly.

I’ve been thinking this would be a really nice improvement on the pre-defined prompts.

I think it’s worth doing research on other prompt libraries to come up with 10-20 actually useful canned prompts. Then we can implement the library with staring prompts rather than just an empty starting point.
